wsi/drm/drmplane.cpp

```diff
@@ -188,6 +188,39 @@ bool DrmPlane::Initialize(uint32_t gpu_fd,
     in_fence_fd_prop_.id = 0;
   }
 
+  // query and store supported modifiers for format, from in_formats
+  // property 
+  uint64_t in_formats_prop_value; 
```

Comment:
We could have in_Formats_prop_value here https://github.com/intel/IA-Hardware-Composer/blob/master/wsi/drm/drmplane.cpp#L41 by calling equivalent "IN_FORMATS" from here:
https://github.com/intel/IA-Hardware-Composer/blob/master/wsi/drm/drmplane.cpp#L185

If in_Formats_prop_value is valid than go further with PropertyBlob

Reply:
done.